core/src/lib/mem_pool.cc

```diff
@@ -284,16 +284,12 @@ void GarbageCollectMemoryPool()
 void CloseMemoryPool()
 {
   struct abufhead *buf, *next;
-  int count = 0;
-  uint64_t bytes = 0;
 
   P(mutex);
   for (int i = 1; i <= PM_MAX; i++) {
     buf = pool_ctl[i].free_buf;
     while (buf) {
       next = buf->next;
-      count++;
-      bytes += SizeofPoolMemory((char*)buf);
       free((char*)buf);
```

Comment:
```suggestion
      free(buf);
```
I don't see how casting to `char*` makes any difference here. Casting `abufhead*` to `void*` should have the same effect as casting to `char*` and then to `void*`, so the explicit cast could be avoided.